python/mxnet/base.py

```diff
@@ -359,3 +361,79 @@ def _as_list(obj):
         return obj
     else:
         return [obj]
+
+
+_OP_NAME_PREFIX_LIST = ['_contrib_', '_linalg_', '_random_', '_sparse_']
+
+
+def _get_op_name_prefix(op_name):
+    """
+    Check whether the given op_name starts with any words in `_OP_NAME_PREFIX_LIST`.
+    If found, return the prefix; else, return an empty string.
+    """
+    for prefix in _OP_NAME_PREFIX_LIST:
+        if op_name.startswith(prefix):
+            return prefix
+    return ""
+
+
+# pylint: enable=too-many-locals, invalid-name
+def _init_op_module(root_namespace, module_name, make_op_func):
```

Comment:
why put under base?

Reply:
I feel it doesn't fit in base either. Can you suggest one place that is accessible to both symbol/op.h and ndarray/op.h?

Reply:
i see. make sense

Reply:
Let's put it in base for now